### PR TITLE
add Supply Chain Bot Tournament to schedule

### PR DESCRIPTION
**Pull request by fgebhart**
* adding a page with a description of the tournament
* adding a reference to the schedule

Note: I could not figure out how to set the duration of the event. We planned to have it from 10:00 - 10:30 UTC on Thursday.

### CODE REVIEW

File: _talks/422.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: talk
+title: "Supply Chain Bot Tournament"
+kind: "Extracurricular"
+speakers: "Null"
+---
+
+Have you ever wondered how supply chain problems can be solved using a data driven approach? Or did you even ask yourself
+'how hard can it be?' while looking for supermarkets with toilet paper and pasta in stock during the hoarding caused by
```
Comment (tacitvenom) [inline _talks/422.md]:
```suggestion
'how hard can it be?' while looking for supermarkets with toilet paper and pasta out of stock during the hoarding caused by
```

File: _talks/422.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Corona?
+This is you chance: Write bots for retailers, wholesalers, distributors and manufactures and show the others that you
```
Comment (tacitvenom) [inline _talks/422.md]:
```suggestion
This is your chance: Write bots for retailers, wholesalers, distributors and manufacturers and show others that you
```

File: _talks/422.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+master the problem of supply chain! Proof the world, your bot is the smartest and become famous!
```
Comment (tacitvenom) [inline _talks/422.md]:
```suggestion
master the problem of supply chain! Prove to the world, your bot is the smartest and become famous!
```

**Review by tacitvenom — APPROVED** [review]
just nit-picking

**Review by sebastianneubauer — APPROVED** [review]

**Pull request merged**